### PR TITLE
File5000name

### PR DESCRIPTION
**Pull request by VitaliyKRS**

### CODE REVIEW

File: HW1/SoftServeWOstl/FileWithoutSTL.cpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+	}
+	fout.close();
+}
+
+int weight(string word) {
+	int count = 0;
+	for (int i = 0; i < strlen(word.c_str()); i++) {
+		if (word[i] == 'a')count++;
+		if (word[i] == 'b')count+=2;
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithoutSTL.cpp]: you are so NOT lazy to write all of this))

File: HW1/SoftServeWOstl/FileWithoutSTL.cpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	cout << "Error open";
+}
+else {
+	for (int i = 0; i < size; ++i) {
+		fout << arr2[i] << " ";
+	}
+
+}
+fout.close(); 
+_getch();
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithoutSTL.cpp]: forget this Windows crap, Qt does this functionality for you

File: HW1/SoftServeWOstl/FileWithoutSTL.cpp (hunk 1 above)
Comment (VitaliyKRS) [reply HW1/SoftServeWOstl/FileWithoutSTL.cpp]: )

File: HW1/SoftServeWOstl/FileWithoutSTL.cpp (hunk 2 above)
Comment (VitaliyKRS) [reply HW1/SoftServeWOstl/FileWithoutSTL.cpp]: i wrote this in visual studio

**Pull request merged**